### PR TITLE
[DOCFIX] Add java11 image doc

### PR DESCRIPTION
**Pull request by ssz1997**
Add doc about `alluxio/alluxio-jdk11` image.

### CODE REVIEW

File: docs/en/deploy/Running-Alluxio-On-Docker.md [hunk 1]
```diff
@@ -287,6 +287,12 @@ Congratulations, you've deployed a basic Dockerized Alluxio cluster! Read on to
 
 ## Advanced Setup
 
+### Launch Alluxio with Java11
+Starting from v2.9.0, Alluxio processes can be launched with Java 11 inside Docker containers. The new docker image, `alluxio/alluxio-jdk11` is available
+in Dockerhub. Java 11 has more GC options and potential performance gain.
```
Comment (ssz1997) [inline docs/en/deploy/Running-Alluxio-On-Docker.md:292]: Do we want to mention the potential performance gain?

**Comment by Xenorith** [comment]
@HelloHorizon please review post merge

**Comment by Xenorith** [comment]
alluxio-bot, merge this please

**Comment by Xenorith** [comment]
alluxio-bot, cherry-pick this to branch-2.9 please

File: docs/en/deploy/Running-Alluxio-On-Docker.md (hunk 1 above)
Comment (HelloHorizon) [reply docs/en/deploy/Running-Alluxio-On-Docker.md:292]: Let's remove this part till we have benchmark result in public.

Comment (Xenorith) [reply docs/en/deploy/Running-Alluxio-On-Docker.md:292]: sounds good, i will update this in a batch of small docfixes

**Pull request merged**